astroid/raw_building.py

```diff
@@ -390,7 +390,7 @@ def object_build(
                 with warnings.catch_warnings():
                     warnings.simplefilter("error")
                     member = getattr(obj, name)
-            except (AttributeError, DeprecationWarning):
+            except (AttributeError, DeprecationWarning, FutureWarning):
                 # damned ExtensionClass.Base, I know you're there !
                 attach_dummy_node(node, name)
                 continue
```

Comment:
I agree with @jacobtylerwalls that we should (try to) change the filtering logic to not escalate to error (it's risky if we miss one escalation, on the top of my head we're missing UserWarning, PendingDeprecationWarning and SyntaxWarning at least).Maybe something like this ?
```python
with warnings.catch_warnings():
    warnings.simplefilter("ignore")
    member = getattr(obj, name)
```    

 I'm wondering what we do in  ``attach_dummy_node(node, name)`` that could necessitate to list everything. But I just checked and we indeed missed a lot of warning, it's actually surprising that this was not reported before : https://docs.python.org/3/library/warnings.html#warning-categories. Are those specific warnings never used in any library pylint astroid is parsing ?